2025-08-18 08:00:05 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:00:05 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:00:20 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:00:20 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:00:20 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:00:20 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:00:20 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:00:20 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:00:35 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:00:35 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:00:50 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:00:50 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:00:50 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:00:50 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:00:50 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:00:50 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:01:05 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:01:05 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:01:20 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:01:20 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:01:20 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:01:20 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:01:21 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:01:21 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:01:35 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:01:35 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:01:50 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:01:50 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:01:50 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:01:50 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:01:51 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:01:51 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:02:05 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:02:05 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:02:20 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:02:20 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:02:20 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:02:20 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:02:21 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:02:21 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:02:35 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:02:35 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:02:50 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:02:50 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:02:50 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:02:50 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:02:51 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:02:51 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:03:05 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:03:05 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:03:20 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:03:20 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:03:20 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:03:20 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:03:21 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:03:21 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:03:35 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:03:35 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:03:51 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:03:51 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:03:51 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:03:51 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:03:51 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:03:51 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:04:06 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:04:06 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:04:21 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:04:21 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:04:21 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:04:21 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:04:21 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:04:21 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:04:36 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:04:36 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:04:51 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:04:51 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:04:51 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:04:51 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:04:51 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:04:51 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:05:06 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:05:06 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:05:21 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:05:21 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:05:21 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:05:21 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:05:21 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:05:21 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:05:36 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:05:36 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:05:51 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:05:51 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:05:51 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:05:51 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:05:51 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:05:51 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:06:06 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:06:06 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:06:21 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:06:21 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:06:21 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:06:21 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:06:22 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:06:22 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:06:36 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:06:36 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:06:51 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:06:51 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:06:51 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:06:51 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:06:52 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:06:52 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:07:06 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:07:06 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:07:21 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:07:21 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:07:21 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:07:21 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:07:22 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:07:22 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:07:36 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:07:36 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:07:51 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:07:51 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:07:51 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:07:51 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:07:52 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:07:52 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [127.0.0.1/<unresolved>:6379]: Connection refused: /127.0.0.1:6379
2025-08-18 08:08:06 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:08:06 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:08:21 [AMQP Connection 127.0.0.1:5672] ERROR c.r.c.impl.ForgivingExceptionHandler - Caught an exception when recovering topology Caught an exception while recovering consumer amq.ctag-hY9FXxLcy-8us4MkIVs3nQ: null
com.rabbitmq.client.TopologyRecoveryException: Caught an exception while recovering consumer amq.ctag-hY9FXxLcy-8us4MkIVs3nQ: null
	at com.rabbitmq.client.impl.recovery.AutorecoveringConnection.recoverConsumer(AutorecoveringConnection.java:868)
	at com.rabbitmq.client.impl.recovery.AutorecoveringConnection.recoverTopology(AutorecoveringConnection.java:732)
	at com.rabbitmq.client.impl.recovery.AutorecoveringConnection.beginAutomaticRecovery(AutorecoveringConnection.java:602)
	at com.rabbitmq.client.impl.recovery.AutorecoveringConnection.lambda$addAutomaticRecoveryListener$3(AutorecoveringConnection.java:524)
	at com.rabbitmq.client.impl.AMQConnection.notifyRecoveryCanBeginListeners(AMQConnection.java:839)
	at com.rabbitmq.client.impl.AMQConnection.doFinalShutdown(AMQConnection.java:816)
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:700)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.io.IOException: null
	at com.rabbitmq.client.impl.AMQChannel.wrap(AMQChannel.java:140)
	at com.rabbitmq.client.impl.AMQChannel.wrap(AMQChannel.java:136)
	at com.rabbitmq.client.impl.ChannelN.basicConsume(ChannelN.java:1408)
	at com.rabbitmq.client.impl.recovery.RecordedConsumer.recover(RecordedConsumer.java:60)
	at com.rabbitmq.client.impl.recovery.AutorecoveringConnection.wrapRetryIfNecessary(AutorecoveringConnection.java:914)
	at com.rabbitmq.client.impl.recovery.AutorecoveringConnection.internalRecoverConsumer(AutorecoveringConnection.java:889)
	at com.rabbitmq.client.impl.recovery.AutorecoveringConnection.recoverConsumer(AutorecoveringConnection.java:864)
	... 7 common frames omitted
Caused by: com.rabbitmq.client.ShutdownSignalException: channel error; protocol method: #method<channel.close>(reply-code=404, reply-text=NOT_FOUND - no queue 'url-shortener' in vhost '/', class-id=60, method-id=20)
	at com.rabbitmq.utility.ValueOrException.getValue(ValueOrException.java:66)
	at com.rabbitmq.utility.BlockingValueOrException.uninterruptibleGetValue(BlockingValueOrException.java:36)
	at com.rabbitmq.client.impl.AMQChannel$BlockingRpcContinuation.getReply(AMQChannel.java:552)
	at com.rabbitmq.client.impl.ChannelN.basicConsume(ChannelN.java:1402)
	... 11 common frames omitted
Caused by: com.rabbitmq.client.ShutdownSignalException: channel error; protocol method: #method<channel.close>(reply-code=404, reply-text=NOT_FOUND - no queue 'url-shortener' in vhost '/', class-id=60, method-id=20)
	at com.rabbitmq.client.impl.ChannelN.asyncShutdown(ChannelN.java:528)
	at com.rabbitmq.client.impl.ChannelN.processAsync(ChannelN.java:349)
	at com.rabbitmq.client.impl.AMQChannel.handleCompleteInboundCommand(AMQChannel.java:193)
	at com.rabbitmq.client.impl.AMQChannel.handleFrame(AMQChannel.java:125)
	at com.rabbitmq.client.impl.AMQConnection.readFrame(AMQConnection.java:761)
	at com.rabbitmq.client.impl.AMQConnection.access$400(AMQConnection.java:48)
	at com.rabbitmq.client.impl.AMQConnection$MainLoop.run(AMQConnection.java:688)
	... 1 common frames omitted
2025-08-18 08:08:21 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:08:21 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:08:21 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:08:21 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:08:22 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was 127.0.0.1/<unresolved>:6379
2025-08-18 08:08:22 [lettuce-nioEventLoop-7-6] INFO  i.l.c.protocol.ReconnectionHandler - Reconnected to 127.0.0.1/<unresolved>:6379
2025-08-18 08:08:36 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:08:36 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:08:52 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:08:52 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:08:52 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:08:52 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:09:07 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:09:07 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:09:22 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:09:22 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:09:22 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:09:22 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:09:37 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:09:37 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:09:52 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:09:52 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:09:52 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:09:52 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:10:07 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:10:07 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:10:22 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:10:22 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:10:22 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:10:22 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:10:37 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:10:37 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:10:52 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:10:52 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:10:52 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:10:52 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:11:07 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:11:07 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:11:22 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:11:22 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:11:22 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:11:22 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:11:37 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:11:37 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:11:52 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:11:52 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:11:52 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:11:52 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:12:07 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:12:07 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:12:22 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:12:22 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:12:22 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:12:22 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:12:37 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:12:37 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:12:52 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:12:52 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:12:52 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:12:52 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:13:07 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:13:07 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:13:22 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:13:22 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:13:22 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:13:22 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:13:37 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:13:37 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:13:53 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:13:53 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:13:53 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:13:53 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:14:08 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:14:08 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:14:23 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:14:23 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:14:23 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:14:23 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:14:38 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:14:38 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:14:53 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:14:53 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:14:53 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:14:53 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:15:08 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:15:08 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:15:23 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:15:23 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:15:23 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:15:23 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:15:38 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:15:38 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:15:53 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:15:53 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:15:53 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:15:53 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:16:08 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:16:08 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:16:23 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:16:23 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:16:23 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:16:23 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:16:38 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:16:38 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:16:53 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:16:53 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:16:53 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:16:53 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:17:08 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:17:08 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:17:23 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:17:23 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:17:23 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:17:23 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:17:38 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:17:38 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:17:53 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:17:53 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:17:53 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:17:53 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:18:08 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:18:08 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:18:23 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:18:23 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:18:23 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:18:23 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:18:38 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:18:38 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:18:54 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:18:54 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:18:54 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:18:54 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:19:09 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:19:09 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:19:24 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:19:24 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:19:24 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:19:24 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:19:39 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:19:39 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:19:54 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:19:54 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:19:54 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:19:54 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:20:09 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:20:09 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:20:24 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:20:24 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:20:24 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:20:24 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:20:39 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:20:39 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:20:54 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:20:54 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:20:54 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:20:54 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:21:09 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:21:09 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:21:24 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:21:24 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:21:24 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:21:24 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:21:39 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:21:39 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:21:54 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:21:54 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:21:54 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:21:54 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:22:09 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:22:09 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:22:24 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:22:24 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:22:24 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:22:24 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:22:39 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:22:39 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:22:54 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:22:54 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:22:54 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:22:54 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:23:09 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:23:09 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:23:24 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:23:24 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:23:24 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:23:24 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:23:39 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:23:39 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:23:55 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:23:55 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:23:55 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:23:55 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:24:10 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:24:10 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:24:25 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:24:25 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:24:25 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:24:25 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:24:40 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:24:40 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:24:55 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:24:55 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:24:55 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:24:55 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:25:10 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:25:10 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:25:25 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:25:25 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:25:25 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:25:25 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:25:40 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:25:40 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:25:55 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:25:55 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:25:55 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:25:55 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:26:10 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:26:10 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:26:25 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:26:25 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:26:25 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:26:25 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:26:40 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:26:40 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:26:55 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:26:55 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:26:55 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:26:55 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:27:10 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:27:10 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:27:25 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:27:25 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:27:25 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:27:25 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:27:40 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:27:40 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:27:55 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:27:55 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:27:55 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:27:55 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:28:10 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:28:10 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:28:25 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:28:25 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:28:25 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:28:25 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:28:40 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:28:40 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:28:56 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:28:56 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:28:56 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:28:56 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:29:11 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:29:11 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:29:26 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:29:26 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:29:26 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:29:26 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:29:41 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:29:41 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:29:56 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:29:56 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:29:56 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:29:56 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:30:11 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:30:11 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:30:26 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:30:26 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:30:26 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:30:26 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:30:41 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:30:41 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:30:56 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:30:56 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:30:56 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:30:56 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:31:11 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:31:11 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:31:26 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:31:26 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:31:26 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:31:26 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:31:41 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:31:41 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:31:56 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:31:56 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:31:56 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:31:56 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:32:11 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:32:11 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:32:26 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:32:26 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:32:26 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:32:26 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:32:41 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:32:41 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:32:56 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:32:56 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:32:56 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:32:56 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:33:11 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:33:11 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:33:26 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:33:26 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:33:26 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:33:26 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:33:41 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:33:41 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:33:57 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:33:57 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:33:57 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:33:57 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:34:12 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:34:12 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:34:27 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:34:27 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:34:27 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:34:27 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:34:42 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:34:42 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:34:57 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:34:57 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:34:57 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:34:57 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:35:12 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:35:12 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:35:27 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:35:27 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:35:27 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:35:27 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:35:42 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:35:42 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:35:57 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:35:57 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:35:57 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:35:57 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:36:12 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:36:12 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:36:27 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:36:27 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:36:27 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:36:27 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:36:42 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:36:42 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:36:57 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:36:57 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:36:57 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:36:57 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:37:12 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:37:12 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:37:27 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:37:27 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:37:27 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:37:27 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:37:42 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:37:42 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:37:57 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:37:57 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:37:57 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:37:57 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:38:12 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:38:12 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:38:27 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:38:27 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:38:27 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:38:27 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:38:42 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:38:42 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:38:57 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:38:57 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:38:57 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:38:57 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:39:12 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:39:12 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:39:28 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:39:28 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:39:28 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:39:28 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:39:43 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:39:43 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:39:58 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:39:58 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:39:58 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:39:58 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:40:13 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:40:13 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:40:28 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:40:28 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:40:28 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:40:28 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:40:43 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:40:43 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:40:58 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:40:58 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:40:58 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:40:58 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:41:13 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:41:13 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:41:28 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:41:28 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:41:28 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:41:28 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:41:43 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:41:43 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:41:58 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:41:58 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:41:58 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:41:58 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:42:13 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:42:13 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:42:28 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:42:28 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:42:28 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:42:28 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:42:43 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:42:43 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:42:58 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:42:58 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:42:58 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:42:58 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:43:13 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:43:13 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:43:28 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:43:28 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:43:28 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:43:28 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:43:43 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:43:43 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:43:58 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:43:58 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:43:58 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:43:58 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:44:13 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:44:13 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:44:29 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:44:29 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:44:29 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:44:29 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:44:44 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:44:44 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:44:59 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:44:59 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:44:59 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:44:59 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:45:14 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:45:14 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:45:29 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:45:29 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:45:29 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:45:29 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:45:44 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:45:44 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:45:59 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:45:59 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:45:59 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:45:59 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:46:14 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:46:14 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:46:29 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:46:29 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:46:29 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:46:29 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:46:44 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:46:44 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:46:59 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:46:59 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:46:59 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:46:59 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:47:14 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:47:14 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:47:29 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:47:29 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:47:29 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:47:29 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:47:44 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:47:44 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:47:59 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:47:59 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:47:59 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:47:59 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:48:14 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:48:14 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:48:29 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:48:29 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:48:29 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:48:29 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:48:44 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:48:44 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:48:59 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:48:59 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:48:59 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:48:59 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:49:14 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:49:14 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:49:30 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:49:30 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:49:30 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:49:30 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:49:45 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:49:45 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:50:00 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:50:00 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:50:00 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:50:00 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:50:15 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:50:15 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:50:30 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:50:30 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:50:30 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:50:30 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:50:45 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:50:45 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:51:00 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:51:00 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:51:00 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:51:00 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:51:15 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:51:15 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:51:30 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:51:30 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:51:30 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:51:30 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:51:45 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:51:45 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:52:00 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:52:00 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:52:00 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:52:00 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:52:15 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:52:15 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:52:30 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:52:30 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:52:30 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:52:30 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:52:45 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:52:45 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:53:00 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:53:00 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:53:00 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:53:00 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:53:15 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:53:15 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:53:30 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:53:30 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:53:30 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:53:30 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:53:45 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:53:45 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:54:00 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:54:00 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:54:00 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:54:00 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:54:15 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:54:15 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:54:31 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:54:31 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:54:31 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:54:31 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:54:46 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:54:46 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:55:01 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:55:01 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:55:01 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:55:01 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:55:16 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:55:16 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:55:31 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:55:31 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:55:31 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:55:31 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:55:46 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:55:46 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:56:01 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:56:01 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:56:01 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:56:01 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:56:16 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:56:16 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:56:31 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:56:31 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:56:31 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:56:31 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:56:46 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:56:46 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:57:01 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:57:01 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:57:01 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:57:01 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:57:16 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:57:16 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:57:31 [lettuce-eventExecutorLoop-3-1] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:57:31 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:57:31 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:57:31 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:57:46 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:57:46 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:58:01 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:58:01 [lettuce-eventExecutorLoop-3-6] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:58:01 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:58:01 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:58:16 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:58:16 [lettuce-nioEventLoop-7-4] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:58:31 [lettuce-eventExecutorLoop-3-3] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:58:31 [lettuce-eventExecutorLoop-3-2] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:58:31 [lettuce-nioEventLoop-7-5] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:58:31 [lettuce-nioEventLoop-7-6] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:58:46 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:58:46 [lettuce-nioEventLoop-7-7] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:59:01 [lettuce-eventExecutorLoop-3-8] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:59:01 [lettuce-eventExecutorLoop-3-9] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:59:01 [lettuce-nioEventLoop-7-8] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:59:01 [lettuce-nioEventLoop-7-9] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:59:16 [lettuce-eventExecutorLoop-3-10] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:59:16 [lettuce-nioEventLoop-7-10] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
2025-08-18 08:59:32 [lettuce-eventExecutorLoop-3-5] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26381
2025-08-18 08:59:32 [lettuce-eventExecutorLoop-3-4] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26379
2025-08-18 08:59:32 [lettuce-nioEventLoop-7-2] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26379]: Connection refused: localhost/127.0.0.1:26379
2025-08-18 08:59:32 [lettuce-nioEventLoop-7-1] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26381]: Connection refused: localhost/127.0.0.1:26381
2025-08-18 08:59:47 [lettuce-eventExecutorLoop-3-7] INFO  i.l.core.protocol.ConnectionWatchdog - Reconnecting, last destination was localhost/<unresolved>:26380
2025-08-18 08:59:47 [lettuce-nioEventLoop-7-3] WARN  i.l.core.protocol.ConnectionWatchdog - Cannot reconnect to [localhost/<unresolved>:26380]: Connection refused: localhost/127.0.0.1:26380
